src/manifest.ts

```diff
@@ -1,12 +1,14 @@
+import type { Manifest } from 'webextension-polyfill';
 import pkg from '../package.json';
-import { ManifestType } from '@src/manifest-type';
 
-const manifest: ManifestType = {
+const manifest: Manifest.WebExtensionManifest = {
   manifest_version: 3,
   name: pkg.displayName,
   version: pkg.version,
   description: pkg.description,
-  options_page: 'src/pages/options/index.html',
```

Comment:
this is deprecated [ref](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_page)

Reply:
Hey @verden11, thank you so much for the contribution!

Will check it out either today or tomorrow and merge if it works.